### PR TITLE
Get rid of the remnants of the old expression normalization support

### PR DESCRIPTION
**Pull request by elprans**
The new expression normalization mechanism that was added in #1522
(0d110e3a75e5) left the support infrastructure for the old system in
place, namely: all the `orig_expr` bits.  This finishes the job by
ripping those bits off.

### CODE REVIEW

File: edb/edgeql/compiler/normalization.py [hunk 1]
```diff
@@ -42,6 +45,46 @@ def normalize(
...
 
 
+def renormalize_compat(
+    norm_qltree: qlast.Base,
+    orig_text: str,
+    *,
+    schema: s_schema.Schema,
+    localnames: AbstractSet[str] = frozenset(),
+) -> qlast.Base:
+    """Renormalize an expression normalized with imprint_expr_context()."""
```
Comment (1st1) [inline edb/edgeql/compiler/normalization.py:55]: Would help if you could clarify in this docstring what renormalization would actually do to it.

Comment (1st1) [reply edb/edgeql/compiler/normalization.py:55]: Looks like this is needed for compatibility with pre a7 dumps?

Comment (1st1) [reply edb/edgeql/compiler/normalization.py:55]: And it essentially re-creates the old WITH-style normalization?

**Review by 1st1 — APPROVED** [review]

File: edb/edgeql/compiler/normalization.py (hunk 1 above)
Comment (elprans) [reply edb/edgeql/compiler/normalization.py:55]: It takes the old `WITH-style` normalization and renormalizes it into the new explicit module name injection.

Comment (1st1) [reply edb/edgeql/compiler/normalization.py:55]: So... can you add that to the docstring?

**Pull request merged**